hr_payroll_commission/README.rst

```diff
@@ -0,0 +1,92 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Payroll Commissions

```

Comment:
This module really computes commissions on invoiced sales.
The title and description should say that.